suite/tests/client-interface/syscall-mod.c

```diff
@@ -45,7 +45,7 @@
 
 int main()
 {
-    int pid;
+    int64_t pid;
```

Comment:
Looks like the X86 inline assembly is not happy with this type. I will update that

Reply:
I changed it back to int and limited the width in the AArch64 assembler.